l10n/en/firefox/whatsnew/whatsnew.ftl

```diff
@@ -24,6 +24,6 @@ whatsnew-release-notes = Read the <a href="{ $url }">Release Notes</a> to know m
 
 whatsnew-signoff = <strong>Powered by { -brand-name-mozilla }.</strong> Putting people before profits since 1998.
 
-whatsnew-donate = Donate to the { -brand-name-mozilla-foundation }
+whatsnew-donate-to-mofo = Donate to the { -brand-name-mozilla-foundation }
```

Comment:
**Issue:** This doesn't match the `ftl_has_messages('whatsnew-donate')` id check: https://github.com/mozilla/bedrock/blob/1a83d88d7cbf6059a27f51f620eb8bb5ab640145/bedrock/firefox/templates/firefox/whatsnew/index.html#L52-L54

Isn't it just leftover from testing how missing translations behave?